Justfile

```diff
@@ -0,0 +1,16 @@
+flox_bats_tests_invocation := "flox run '.#flox-tests' -- -- --flox target/debug/flox"
+
+_default:
+    @just --list --unsorted
+
+# Run the 'bats' test suite
+shell-test +test_files="":
```

Comment:
Any of `integration`/`integration-test`/`bats-test` would be clearer to me than `shell-test`

Reply:
Sure. I've also come around to `test`->`tests` since submitting the PR.